extensions/podman/scripts/download.ts

```diff
@@ -20,6 +20,11 @@
 import { PodmanDownload } from './podman-download';
 
 import * as podman4JSON from '../src/podman4.json';
+import * as podman5JSON from '../src/podman5.json';
 
 const podman4Download = new PodmanDownload(podman4JSON, true);
 podman4Download.downloadBinaries();
+
+// do not fetch for aigap mode now
```

Comment:
typo: airgap (too much ai on the brain :) )

Reply:
Thanks 😂